docs/platform/howto/create-service-integration.rst

```diff
@@ -0,0 +1,30 @@
+Create service integration
+===========================
+
+If you're not sure what service integration is, follow :doc:`this guide <../concepts/service-integration>` for an overview. This article will guide you on how to create service integrations between different Aiven services and move telemetry data using these integrations. To get started, you will need three services:
+
+- Aiven for Apache Kafka® - The service that produces the telemetry data
+- Aiven for PostgreSQL® - Database where the telemetry data is stored and can be queried from
+- Aiven for Grafana® - Dashboards for the telemetry data
+
+.. note::
+
+    Currently these services need to be running in the same Aiven project, but this limitation will be removed in the future and you will be able to, for example, connect multiple Aiven for Apache Kafka® services from different projects to the same PostgreSQL® and Grafana® instances.
+
+Steps:
+
+1. From the Aiven console, follow :doc:`this guide <create_new_service>` to create three new services - Aiven for Apache Kafka®, Aiven for PostgreSQL®, and Aiven for Grafana®. You can choose your preferred cloud provider, region, and any plan from **startup** / **business** / **premium**.  
+
+2. Once all three services are running, navigate to the PostgreSQL service on the Aiven console and click on **Service integrations** from the **Overview** page. Click **Use integration** button for the *Grafana Metrics Dashboard* card. The **Existing service** radio button should be highlighted next and you can choose the newly create Grafana service.
+Next, click **Enable**. 
```

Comment:
```suggestion
   Next, click **Enable**. 
```
Either add leading spaces, or merge into the previous line, to make the list item display propely.